pkg/kubelet/config/common.go

```diff
@@ -43,6 +43,10 @@ import (
 	"k8s.io/klog"
 )
 
+const (
+	maxReadLength = 1 << 20
```

Comment:
Nit: Could you leave a comment that is something along the lines of `1 << 20 is 1MB`. Maybe that's obvious to other people haha, but I had to look it up :)